Block/Customer/Cards.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Omise\Payment\Block\Customer;
+
+use Magento\Framework\View\Element\Template;
+use Omise\Payment\Model\Customer;
+
+/**
+ * Class Cards
+ */
+class Cards extends Template
+{
+    /**
+     * @var Omise\Payment\Model\Customer;
+     */
+    private $customer;
+
+    /**
+     * @param \Magento\Framework\View\Element\Template $context
+     * @param Omise\Payment\Model\Customer $customer
+     * @param array $data
+     */
+    public function __construct(
+        Template\Context $context,
+        Customer $customer,
+        array $data = []
+    ) {
+        parent::__construct($context, $data);
+        $this->customer = $customer;
+    }
+
+    /**
+     * @return  array|null
+     */
+    public function getCards()
+    {
+        return !$this->customer->getMagentoCustomerId() || !$this->customer->getId() ? null : $this->customer->cards(['order' => 'reverse_chronological']);
```

Comment:
`reverse_chronological` feels like it should be a constant

Reply:
If you search for that in magento code, they use that as string, we do too. This is better as `constant` but better would be to do it as seperated PR and find proper place to put that constant, and replace that in other places.

Reply:
OK, if they're doing it that way